lib/Convos/Core/Backend.pm

```diff
@@ -75,7 +75,7 @@ sub _setup {
           my ($connection, $target, $msg) = @_;
 
           if ($msg->{highlight} and $target->id and !$target->is_private) {
-            $connection->user->{unread}++;
+            $target->inc_notifications_p->catch(sub { $self->_debug('inc_notifications %s FAIL %s', $target->id, shift) } );
```

Comment:
Since the class doesn't really do anything, you can change this to inc_notifications() instead:

```
sub inc_notifications { $_[0]->{notifications}++; $_[0] }
```